Pagee.html

```diff
@@ -138,22 +152,28 @@
         <div class="calculator_container">
             <div class="container">
```

Comment:
why do you need this container? 
1. container it's not meaningful
2. container in container? 
 <div class="calculator_container">
            <div class="container">

Reply:
because i need to divide this calculator for 2 parts
inputs part and result part